src/components/NumberInput/config.js

```diff
@@ -0,0 +1,13 @@
+let NumberInputConfig = {
+  DEFAULT_FONT_SIZE: 40,
+  POPUP_X: 0,
+  POPUP_Y: 0,
+  POPUP_WIDTH: 80,
+  POPUP_HEIGHT: 80,
+  POPUP_COLOR: 0xFF0eb4ea,
```

Comment:
Colors are defined in theme/default.js please use this for color definitions

Reply:
POPUP_COLOR: 0xFF0eb4ea, color config removed from config file.